ncm-metaconfig/src/main/metaconfig/limits_conf/tests/regexps/config

```diff
@@ -0,0 +1,9 @@
+Base test for limitsconfig
+---
+/etc/security/limits.d/99-quattor.conf
+---
+^ftp\s{4}hard\s{4}priority\s{4}5$
+^student\s{4}soft\s{4}maxlogins\s{4}1$
+^ceph\s{4}-\s{4}nproc\s{4}1048576$
```

Comment:
don't tell me we use this for ceph? on el7, all service limits should be configured via the unit limits